Modules/DatabricksPS/Public/General.ps1

```diff
@@ -425,7 +425,11 @@ Function Set-DatabricksEnvironment {
 				$script:dbApiRootUrl = "https://$($azResource.Properties.workspaceUrl.Trim('/'))/api"
 
 				$azToken = Get-AzAccessToken -ResourceUrl "2ff814a6-3304-4ab8-85cb-cd0e6f879c1d"
-				$script:dbAuthenticationHeader["Authorization"] = "$($azToken.Type) $($azToken.Token)"
+				$plainToken = $azToken.Token
+				if ($plainToken -is [System.Security.SecureString]) {
+					$plainToken = [System.Net.NetworkCredential]::new('', $plainToken).Password
```

Comment:
I'd rather use this code which is more "powershell-ish"

`$plainToken = ConvertFrom-SecureString -SecureString $plainToken -AsPlainText`